README.md

```diff
@@ -16,6 +16,15 @@ It's primarily tested on newer versions of Ubuntu, but should work on both Debia
 
 go::server will install and start an empty Go server.
 
+Before running chef-client with this recipe , ensure you have the
+following setting on your ci-server else Go server start will fail
+
+$ hostname 
+<server-hostname>
+
+$ vim /etc/hosts
+127.0.0.1 <server-hostname>
```

Comment:
You can simplify this to:
```
echo 127.0.0.1 `hostname` | tee -a /etc/hosts
```

Doing that requires no user input, and is a bit quicker to complete.